xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/reference/XWikiSyntaxResourceRenderer.java

```diff
@@ -76,7 +76,7 @@ public String serialize(ResourceReference reference, boolean isFreeStanding)
         return result;
     }
 
-    public void beginRenderLink(XWikiSyntaxEscapeWikiPrinter printer, ResourceReference reference,
+    public void beginRenderLink(XWikiSyntaxEscapeWikiPrinter printer,
```

Comment:
You also need to fix any code calling this method. You should always build the module you are modifying with Maven before committing. If you are using the XWiki dev VM just execute `mvn clean install -Pquality` in the parent folder containing the pom.xml file. If you don't use the VM see http://dev.xwiki.org/xwiki/bin/view/Community/Building.